src/js/components/FileInput/stories/Custom.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { Box, Grommet, FileInput, Text } from 'grommet';
+import { Trash } from 'grommet-icons';
+
+const customTheme = {
+  fileInput: {
+    background: '#f2f2f2',
+    border: { size: 'medium' },
+    pad: { horizontal: 'large', vertical: 'medium' },
+    round: 'small',
+    label: {
+      size: 'large',
+    },
+    icons: {
+      remove: Trash,
```

Comment:
When I drag a file and try to click the trash can, it's not a button that deletes the file. instead, it just opens the file browser again.

Reply:
Are you sure you're clicking the Trash can? There is a Button with the trash can icon and then a input is rendered to the left of the Button.